Components/Button/button.css

```diff
@@ -0,0 +1,411 @@
+.button-container
+{
+    width: 100%;
+    height: fit-content;
+    box-shadow: rgba(0, 0, 0, 0.02) 0px 1px 3px 0px, rgba(27, 31, 35, 0.15) 0px 0px 0px 1px;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    flex-wrap: wrap;
+    padding: 3rem 1rem;
+}
+
+
+/* Primary Solid Buttons */
+.solid-primary-btn
+{
+    min-width: 120px;
+    min-height: 45px;
```

Comment:
Better to give lengths in rems. Rest all looks good.

Reply:
Done, Thanks for review.